axum-extra/Cargo.toml

```diff
@@ -113,6 +112,7 @@ serde_path_to_error = { version = "0.1.8", optional = true }
 tokio = { version = "1.19", optional = true }
 tokio-stream = { version = "0.1.9", optional = true }
 tokio-util = { version = "0.7", optional = true }
+tower = { version = "0.5.2", default-features = false, features = ["util"], optional = true }
```

Comment:
Can you move this down to the "doc dependencies", next to serde? ^^'

Reply:
Updated.